res/controllers/Numark Mixtrack Pro.midi.xml

```diff
@@ -14,6 +61,7 @@
             <control>
                 <group>[Channel1]</group>
                 <key>volume</key>
+                <description>[Channel1], volume</description>
```

Comment:
What do you consider this useful for?
In the mapping GUI the controls are already parsed to human-readable string, and for the description some like "mixer" is useful so the controls can be sorted by category.

Also, the commit message doesn't match the change set, settings are added